pom.xml

```diff
@@ -114,16 +138,6 @@
               </execution>
             </executions>
           </plugin>
-          <plugin>
-            <groupId>org.sonatype.plugins</groupId>
-            <artifactId>nexus-staging-maven-plugin</artifactId>
-            <extensions>true</extensions>
-            <configuration>
-              <serverId>rso</serverId>
-              <nexusUrl>https://repository.sonatype.org/</nexusUrl>
-              <autoReleaseAfterClose>false</autoReleaseAfterClose>
-            </configuration>
-          </plugin>
         </plugins>
       </build>
```

Comment:
In R we also have maybe it is also necessary to add it for the helm. @bhamail  @allenhsieh  ?
      <properties>
        <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
      </properties>

Reply:
@aornatovskyy I am so sorry for the delayed response, we've been working on AuditJS so furiously that a notification was lost in my inbox. Final answer should come from Dan but from what I understand because this parameter exists in `.maven.xml`: https://github.com/sonatype-nexus-community/nexus-repository-helm/blob/trinityneworb/.circleci/.maven.xml#L20 -- it is not also needed in the pom.xml